.github/workflows/main.yml

```diff
@@ -11,17 +11,17 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: 14
-          
+          node-version: 16
```

Comment:
Is it necessary to do it in this oversized PR?

Reply:
Old version doesnt seem to find @chainlink/contracts when installing packages and throws this error: 

`Cannot read property '@chainlink/contracts' of undefined`